packages/next-codemod/transforms/name-default-component.ts

```diff
@@ -1,56 +1,61 @@
 import { basename, extname } from 'path'
 
-const camelCase = (value) => {
-  const val = value.replace(/[-_\s.]+(.)?/g, (_match, chr) =>
+const camelCase = (value: string): string => {
+  const val: string = value.replace(/[-_\s.]+(.)?/g, (_match, chr) =>
     chr ? chr.toUpperCase() : ''
   )
   return val.substr(0, 1).toUpperCase() + val.substr(1)
 }
 
-const isValidIdentifier = (value) => /^[a-zA-ZÀ-ÿ][0-9a-zA-ZÀ-ÿ]+$/.test(value)
+const isValidIdentifier = (value: string): boolean =>
+  /^[a-zA-ZÀ-ÿ][0-9a-zA-ZÀ-ÿ]+$/.test(value)
+
+type Node = {
+  type: string
+  declaration: {
+    type: string
+    body: any
+    id?: any
+  }
+}
 
 export default function transformer(file, api, options) {
   const j = api.jscodeshift
   const root = j(file.source)
 
-  let hasModifications
+  let hasModifications: boolean
 
-  const returnsJSX = (node) =>
+  const isJSX = (node): boolean =>
```

Comment:
Change variable name for readability.

Reply:
`returnsJSX` is correct here. Note that `return` is searched for first: `.find(j.ReturnStatement)`

Reply:
I see ! Thank you !